packages/react-jsx-babel-plugin/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-jsx-babel-plugin",
+  "version": "1.0.0",
```

Comment:
Let's start with 0.1.0 to signal this isn't a thing people should use yet? Also we need to decide whether to mark it as `private` until we want others to try it.

`babel-plugin` shouldn't be the suffix. It should be a prefix: https://babeljs.io/docs/en/plugins/#plugin-shorthand. But we'll probably want to do it in upstream Babel plugin anyway — this is only for testing, right? In that case it should be `private`.

Reply:
Yes, this is only for testing purposes. When we publish the changes it will be to the actual Babel plugin. I'll mark this test version as private.